.gitignore

```diff
@@ -21,3 +21,5 @@ erl_crash.dump
 phoenix-*.ez
 
 .DS_Store
+
+/.vscode/
```

Comment:
These changes should go to your global gitignore: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Allegedly the .DS_Store one too :D